fil_fungible_token/src/token/mod.rs

```diff
@@ -112,7 +116,8 @@ where
         operator: &Address,
         initial_owner: &Address,
         amount: &TokenAmount,
-        data: &RawBytes,
+        operator_data: RawBytes,
+        token_data: RawBytes,
```

Comment:
It was intentional but I'm not sure if my reasoning is too pedantic or even idiomatic with rust conventions:

The addresses and token amounts are borrowed because the method doesn't really "operate on" them and only reads their values. 

I think of the data as consumed by the method, which needs to "send" it. My intuition for this comes from the fact that constructing the `TokensReceivedParams` now can simply consume the moved value rather than cloning from the reference which seems neater and more idiomatic with my understanding of borrowing vs ownership. 

Counterpoint is that we often clone from references to `TokenAmount` within this library. However, I tend to think of the TokenAmount as a primitive and clone as necessary wherever a primitive would have been copied.

Reply:
Makes sense, thanks.